### DISCUSSION TITLE
Welcome to infinite-autocraft Discussions!

### DISCUSSION THREAD

**Discussion by szdytom** [szdytom/infinite-autocraft#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Supyovalk** [comment]
Okay so in the "Known Recipes That Craft [X]", the third line of craft result is unnecessary.

**Comment by PythonProgGamer** [comment]
How often do the known crafting recipes get updated?

**Reply by szdytom** [reply]
Whenever needed, important progress or at least 500 new elements

**Reply by coindxter** [reply]
Are you updating it manually or is it automatic?

**Reply by szdytom** [reply]
I used use the automatic scripts in this repo, but cloudflare is getting and more annoying these days.

**Comment by Amogus5612** [comment]
how do i start the script
